etc/kayobe/kolla.yml

```diff
@@ -134,27 +130,24 @@ kolla_sources:
     type: git
     location: https://github.com/stackhpc/stackhpc-inspector-plugins.git
     reference: 1.3.0
-  # Not yet merged: https://github.com/stackhpc/magnum/pull/139
   magnum-base:
     type: git
     location: https://github.com/stackhpc/magnum.git
     reference: stackhpc/{{ openstack_release }}
-  neutron-base:
-    type: git
-    location: https://github.com/stackhpc/neutron.git
-    reference: stackhpc/{{ openstack_release }}
   neutron-base-plugin-networking-generic-switch:
     type: git
     location: https://github.com/stackhpc/networking-generic-switch.git
     reference: stackhpc/{{ openstack_release }}
-  octavia-api-plugin-ovn-octavia-provider:
-    type: git
-    location: https://github.com/stackhpc/ovn-octavia-provider.git
-    reference: stackhpc/{{ openstack_release }}
-  octavia-driver-agent-plugin-ovn-octavia-provider:
-    type: git
-    location: https://github.com/stackhpc/ovn-octavia-provider.git
-    reference: stackhpc/{{ openstack_release }}
+  prometheus-memcached-exporter:
+    version: 0.14.4
+    sha256:
+      amd64: 60b0fb90e55937869e2a7aa1310d148d56a28639ca3373e0c50548ff8eebaaa1
+      arm64: 053c6020bb2321d301bb1d43fbfc5aec64c3104a45a94338b4ca9351fc9ee014
+  prometheus-v2-server:
+    version: 2.53.1
+    sha256:
+      amd64: 2234aa0f66d9f9b854144f6faaaed72a316df7a680d9dad7cb48e49a6fa5332c
+      arm64: a7f28c83c3c943b953a9d00860bd3f2422464fb7c27a3c4037ef1ce2a41592b5
```

Comment:
Does this actually work? I tried it but I couldn't verify that they are actually built with those versions from build log.

Reply:
Maybe not :thinking: 
```
(prometheus-server)[prometheus@rocky-ovn-fqdn-controller-02 /prometheus-server]$ ./prometheus --version
prometheus, version 2.50.1 (branch: HEAD, revision: 8c9b0285360a0b6288d76214a75ce3025bce4050)
```

Reply:
As discussed, there is a dependency on an upstream fix. I've reverted this change and we can fix it in another PR